src/ast_function_definition.cpp

```diff
@@ -1,13 +1,38 @@
 #include "ast_function_definition.hpp"
 
-FunctionDefinition::FunctionDefinition(Node* declaration_specifiers, Node* declarator, Node* compound_statement) {
-  branches.insert(branches.end(), {declaration_specifiers, declarator, compound_statement});
+void FunctionDefinition::emitRISC(std::ostream &stream, Context &context) const
+{
+    // TODO: this is not complete.
+    std::cerr << "FunctionDefinition: emitRISC is not fully implemented." << std::endl;
```

Comment:
If we're putting this reminder, shouldn't it appear in every single `emitRISC` function we provide?